usrtools/txgen/app/cli-functions.c

```diff
@@ -276,14 +347,21 @@ pcap_cmd(int argc, char **argv)
     return 0;
 }
 
-static struct cli_map start_map[] = {{10, "start %P"}, {20, "stop %P"}, {-1, NULL}};
+static struct cli_map start_map[] = {{10, "start %P"},
+                                     {20, "stop %P"},
+                                     {50, "start %P latsampler"},
```

Comment:
Could add %|latsampler|lat|latency if you like. Just makes it simpler to remember

Reply:
will do